src/components/interfaces/MOBILE_API.xml

```diff
@@ -542,6 +542,7 @@
         <element name="VEHICLEDATA_FUELRANGE" since="5.0" />
         <element name="VEHICLEDATA_ENGINEOILLIFE" since="5.0" />
         <element name="VEHICLEDATA_ELECTRONICPARKBRAKESTATUS" since="5.0" />
+        <element name="VEHICLEDATA_CLOUDAPPVEHICLEID" since="5.1"/>
```

Comment:
You are going to need to change the API version at the top of the file in order to use any of this. Don't worry about the date quite yet.